juniper_official/chassis/check-psm-power-usage-state.rule

```diff
@@ -150,6 +150,10 @@ healthbot {
                 }				
                 type string;
                 description "State of power supply module";
+                enumerate __default__ as -1;				
+                enumerate Offline as 0;
+                enumerate Fault as 0;				
```

Comment:
This should be some other value.. This can be as below. Which means for rest of the rules also we should change to __default__ value to -99

__default__ - -99
Fault - -1
Offline - 0
online - 1